ercc/Makefile

```diff
@@ -23,7 +17,7 @@ IAS_GO_DEFINES+=-DUSE_SGX_HARDWARE_MODE
 endif
 
 TEMPLATE_SUFFIX=.template
-GO_TARGETS=provider.go
```

Comment:
Wouldn't it be nicer/cleaner to outsource the template generation also to the `./attestation` subdirectory and a corresponding makefile?